model/adapter.py

```diff
@@ -723,6 +725,32 @@ def query_fun(rxn):
             reaction.bounds = 0, upper_bound
         return reaction
 
+    def minimize_distance(self):
+        """Replaces fluxomics measurements with the minimized distance"""
+        reaction_measurements = [m for m in self.measurements if m['type'] == 'reaction']
```

Comment:
I know most Pythonistas try to avoid for loops but in this case you could really loop once rather than three times.

```python
index = list()
observations = list()
uncertainties = list()
for measure in (m for m in self.measuresments if m['type'] == 'reaction'):
    index.append(m['id'])
    observations.append(np.nanmean(m['measurements']))
    if len(m['measurements']) > 2:
        uncertainties.append(np.nanstd(m['measurements'], ddof=1))
    else:
        uncertainties.append(1.0)
```
and after potentially adding the growth rate:
```python
observations = pd.Series(index=index, data=observations)
```